ec2system/ec2machine.go

```diff
@@ -331,6 +345,18 @@ func readSshAgentKeys() []string {
 	return keyStrings
 }
 
+func (s *System) getEC2Keys() ([]string, error) {
```

Comment:
should this be named getEC2KeyNames to make it clear that this only retrieves names?